src/main/type-definitions/core.d.ts

```diff
@@ -9,6 +9,7 @@
 /// <reference types="jquery" />
 /// <reference types="jqueryui" />
 /// <reference types="moment-timezone" />
+/// <reference types="print-js" />
```

Comment:
@melloware Using declarations from other libraries is pretty confusing at time, and I had to look this up too, but basically : Referencing print-js like this doesn't work because it declares itself as a module, not as a global library (it contains a `export = printJS` statement). What we need to do is to add the global export ourself. Can you please delete this line and add the following (somewhere before `declare namespace PrimeFaces {`)

```typescript
 /**
  * Exposes the print-js library as a global variable. Normally you do not
  * have to use this youself, as PrimeFaces calls it for you when you use
  * the printer component. Also note that while PrimeFaces currently uses
  * this library for printing, this is subject to change in future releases.
  */
 declare const printJS: typeof import("print-js");
```

Reply:
Thank you I will!  Much appreciated for the review

Reply:
Changes Made!